robot_controllers/include/robot_controllers/diff_drive_base.h

```diff
@@ -169,15 +169,16 @@ class DiffDriveBaseController : public Controller
   ros::Time last_update_;
   ros::Duration timeout_;
 
+  // The odom mutex protects the following variables
   boost::mutex odom_mutex_;
   nav_msgs::Odometry odom_;
+  geometry_msgs::Twist limited_cmd_vel_;
+
   ros::Publisher odom_pub_;
   ros::Publisher params_pub_;
   ros::Timer odom_timer_;
   ros::Subscriber cmd_sub_, scan_sub_;
-
-  bool publish_limited_cmd_;
-  ros::Publisher limited_cmd_pub_;
+  ros::Publisher limited_cmd_vel_pub_;
```

Comment:
I would remove the "vel" and just call it limited_cmd_pub_ for consistency with the other params